app/models/service.rb

```diff
@@ -1,7 +1,8 @@
 class Service < ApplicationRecord
   belongs_to :user
   has_many :bookings, dependent: :destroy
-
+  has_one_attached :photo
```

Comment:
Im thinking once we build everything, we can add to the backlog to change this to has many so we can have carousel images in the service index

Reply:
Good point. That's exactly what I'm doing now for the current Service Show view now.